src/components/InputField/component.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
-import { Form, Input } from 'antd'
+import { Form, Input as AntInput } from 'antd'
 import PropTypes from 'prop-types'
 
 const InputField = ({
-  placeholder, validateStatus, help, field, prefix: Prefix, ...inputProps
+  placeholder, validateStatus, inputAs: Input, help, field, prefix: Prefix, ...inputProps
```

Comment:
inputAs: Input -> as: Component

Reply:
prefix: Prefix -> prefix

Reply:
inputProps -> props

Reply:
> inputAs: Input -> as: Component

@VladWithCookies I can't use `as` because formik field has the same props